ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor

```diff
@@ -0,0 +1,22 @@
+﻿@namespace ExtraDry.Blazor
+
+<div style="display: none" data-initial-level="@InitialHierarchyLevel" />
+
+@code {
+
+    [Parameter]
+    public int? InitialHierarchyLevel { get; set; }
+
+    /// <inheritdoc cref="DryPageQueryView.PageQueryBuilder" />
+    [CascadingParameter]
+    public QueryBuilder? QueryBuilder { get; set; }
+
+    protected override void OnParametersSet()
+    {
+        base.OnParametersSet();
+        if (InitialHierarchyLevel.HasValue)
+        {
+            QueryBuilder?.Level.SetInitialLevel(InitialHierarchyLevel.Value);
```

Comment:
Null chaining here seems incomplete, s/b `QueryBuilder?.Level?.SetInitialLevel(...)`.  This should have been caught by code analysis, did I get this wrong?